package.json

```diff
@@ -7,7 +7,8 @@
     "test:open": "npm run lint && cypress open",
     "lint": "eslint ./cypress/e2e/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
-    "test": "cypress run"
+    "test": "cypress open"
```

Comment:
Why did you change this?
Remember is better to have this script as headless for the CI

As well, i don't see the changes from the cy.ts files

Reply:
yep, it should be cypress run intead of cypress open